packages/@orchidui-vue/src/Builder/DataTable/OcDataTable.stories.js

```diff
@@ -34,21 +36,26 @@ export const Default = {
       Button,
     },
     setup() {
+      const changedFields = ref([]);
       const updateFilterData = (data) => {
         // get new data
         args.filter = data;
       };
       const onClickRow = (val) => {
         console.log("onClickRow  ", val);
       };
-      return { args, updateFilterData, onClickRow };
+
+      return { args, updateFilterData, onClickRow, changedFields };
     },
     template: `
           <Theme class="p-8">
             <div>
               <ul>
                 <li v-for="(item, key) in args.filter">{{ key }} : {{ item }}</li>
               </ul>
+              <div>
+                Filter fields changed: {{ changedFields }}
```

Comment:
@HoaiLee filter field is changed, what is this for? why not use the filter model v-bind:filter and @update:filter=, is there a problem in the dashboard.

Reply:
oh isee, for tracker